src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -58,4 +62,32 @@ def get(self, svc_id):
 
         daemon['rgw_metadata'] = metadata
         daemon['rgw_status'] = status
+
         return daemon
+
+
+@ApiController('rgw/proxy')
```

Comment:
With the merge of #21239 the way we specify the controller paths has changed a bit. In #21239 description is an example of how to declare a controller that acts as a proxy. Please update this controller.

Reply:
Done